homeassistant/components/advantage_air/climate.py

```diff
@@ -151,7 +139,7 @@ def hvac_mode(self) -> HVACMode | None:
     @property
     def fan_mode(self) -> str | None:
         """Return the current fan modes."""
-        return ADVANTAGE_AIR_FAN_MODES.get(self._ac["fan"])
+        return FAN_AUTO if self._ac["fan"] == ADVANTAGE_AIR_MYFAN else self._ac["fan"]
```

Comment:
Does `self._ac["fan"]` return `None` if the fan is turned off?

Reply:
I wonder if we shouldn't keep `ADVANTAGE_AIR_FAN_MODES` for being transparent between Ha modes and library modes even if it's strictly not nesseccary?

Reply:
You cannot turn the fan off, you can turn the HVAC off, but the fan mode is always set.

I certainly could keep ADVANTAGE_AIR_FAN_MODES, but that seems redundant given the values are identical.